src/kernel/bitcoinkernel.h

```diff
@@ -1184,6 +1238,11 @@ BITCOINKERNEL_API void btck_block_destroy(btck_Block* block);
  */
 ///@{
 
+/**
+ * Create a new block validation state.
+ */
+BITCOINKERNEL_API btck_BlockValidationState* btck_block_validation_state_create();
```

Comment:
This (and `btck_block_validation_state_copy` needs `BITCOINKERNEL_WARN_UNUSED_RESULT`